aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -239,6 +241,128 @@ Tensor wrapped_scalar_tensor(
 
 } // anonymous namespace
 
+Tensor& isin_out(const Tensor& elements, const Tensor& test_elements, bool assume_unique, bool invert, Tensor& out) {
+  out.resize_as_(elements);
+  if (elements.numel() == 0) {
+    return out;
+  }
+
+  // Condition taken from numpy's implementation.
+  Tensor test_elements_flat = test_elements.ravel();
+  if (test_elements_flat.numel() < static_cast<int64_t>(
+        10.0f * std::pow(static_cast<double>(elements.numel()), 0.145))) {
+    auto iter = TensorIteratorConfig()
+      .set_check_mem_overlap(false)
```

Comment:
I think we should check the overlap since if the tensors within tensor iterator have overlap, then it is not predictable for the result. We should prevent that.